scraper/tests/test_providers.py

```
@@ -161,6 +163,12 @@ def test_fallback_on_empty_html(self):
             assert m.price_out == fb[1]
             assert m.scrape_status == "fallback"
 
+    def test_does_not_fetch_network(self):
+        """ライブ抽出無効の回帰防止: スクレイプ時にページ取得（get_page_text）を呼ばない。"""
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)